src/server/pegasus_server_impl.cpp

```diff
@@ -1667,7 +1667,18 @@ dsn::error_code pegasus_server_impl::start(int argc, char **argv)
             // only be initialized with default values when calling 'LoadLatestOptions', see
             // 'rocksdb/utilities/options_util.h'.
             reset_rocksdb_options(loaded_data_cf_opts, &_table_data_cf_opts);
-            _db_opts.allow_ingest_behind = parse_allow_ingest_behind(envs);
+
+            // here loaded_db_opt should be read success
+            if (envs.size() == 0) {
+                // for reopen db during load balance learning
+                _db_opts.allow_ingest_behind = loaded_db_opt.allow_ingest_behind;
+                LOG_INFO_PREFIX("reopen replica,last_allow_ingest_behind = {}",
+                                loaded_db_opt.allow_ingest_behind);
+            } else {
+                _db_opts.allow_ingest_behind = parse_allow_ingest_behind(envs);
```

Comment:
Would you mind to refactor the code to encapsulate configs updating of `_db_opts` to `reset_rocksdb_options()` as well?

Reply:
I refactor these codes to reset_rocksdb_options now.